src/quo2/components/numbered_keyboard/keyboard_key/view.cljs

```diff
@@ -15,7 +15,7 @@
         [rn/pressable
          {:accessibility-label :keyboard-key
           :disabled            disabled?
-          :on-press            on-press
+          :on-press            (fn [] (on-press label))
```

Comment:
I have no preference but  `:on-press #(on-press label)` should also work here 👍

Reply:
Done